tests/dummy/app/components/date-time-test/component.js

```diff
@@ -2,5 +2,6 @@ import Component from '@ember/component';
 import Validations from './validations';
 
 export default Component.extend(Validations, {
-  date: null,
+  value: null,
```

Comment:
if you change this to `value`, you would ideally need to change the validator. Probably easier to change back to `date`